lib/internal/modules/run_main.js

```diff
@@ -66,9 +82,12 @@ async function handleMainPromise(promise) {
   }
 }
 
-// For backwards compatibility, we have to run a bunch of
-// monkey-patchable code that belongs to the CJS loader (exposed by
-// `require('module')`) even when the entry point is ESM.
+/**
+ * Parse the CLI main entry point string and run it.
+ * For backwards compatibility, we have to run a bunch of monkey-patchable code that belongs to the CJS loader (exposed
+ * by `require('module')`) even when the entry point is ESM.
+ * @param {string} main CLI main entry point string
```

Comment:
The default value of this parameter is missing from the JSDoc.

Reply:
It would just be repeating what’s in the JavaScript. I’m avoiding specifying anything that the TypeScript Language Server can infer.